test/models/ToolpadEditor.ts

```diff
@@ -75,6 +77,8 @@ export class ToolpadEditor {
     this.componentCatalog = page.getByTestId('component-catalog');
     this.componentEditor = page.getByTestId('component-editor');
 
+    this.themeEditor = page.getByTestId('theme-editor');
```

Comment:
I meant to not test the theme editor, but ok 😄

Reply:
Oh, you meant just starting with dark theme... I thought you meant testing light/dark theme was ok but not UI colors as those could change more later.
Anyway even if we change things later I guess some of these could still be useful, if not we'll eventually delete some of this and that's fine.